tools/pipelines/templates/build-npm-package.yml

```diff
@@ -378,7 +378,8 @@ stages:
               and(
                 succeeded(),
                 eq(variables['Build.Reason'], 'PullRequest'),
-                ne(variables['System.PullRequest.IsFork'], 'true')
+                ne(variables['System.PullRequest.IsFork'], 'true'),
+                false
```

Comment:
One of your comments talks about "disabling coverage" which I thought was test coverage but I guess you meant this, right?

Reply:
Yep